timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -618,8 +626,27 @@ export class TimeGraphChart extends TimeGraphChartLayer {
             if (el instanceof TimeGraphStateComponent && !this.mousePanning && !this.mouseZooming) {
                 this.selectState(el.model);
             }
-            if (this.mouseInteractions && this.mouseInteractions.click) {
+            
+            var self = this;
+            this._mouseClicks++;
+
+            if (this._mouseClicks === 1) {
+                this._recentlyClickedGlobal = cloneDeep(e.data.global);
+                this._dblClickTimer = setTimeout(function(){
+                    self._mouseClicks = 0;
+                    self._recentlyClickedGlobal = null;
+                }, this._dbClickTime);
+            } else if (this._recentlyClickedGlobal && (this._recentlyClickedGlobal.equals(e.data.global))){
+                clearTimeout(this._dblClickTimer);
+                e.type = 'doubleClick';
```

Comment:
Just checking, is it OK to put (our own) type value that is not in the PIXI InteractionEvent.type API?

Be careful, if we send both click and doubleClick as proposed, the type should be ´click' in the click. Maybe better to use the count as condition to detect doubleClick?